perf.groovy

```diff
@@ -0,0 +1,76 @@
+// Groovy Script: http://www.groovy-lang.org/syntax.html
+// Jenkins DSL: https://github.com/jenkinsci/job-dsl-plugin/wiki
+
+import jobs.generation.*;
```

Comment:
@jaredpar This is so we can put the perf jobs on a different server (which will have the perf machines attached to it)